tox.ini

```diff
@@ -12,7 +12,7 @@ exclude = migrations,node_modules
 
 [testenv]
 install_command = pip install -e ".[testing]" -U {opts} {packages}
-commands = coverage run testmanage.py test --deprecation all
+commands = coverage run testmanage.py test --deprecation all {posargs: -v 2}
```

Comment:
This allows running specific tests locally. e.g.

`tox -e python3.9-django3.2-wagtail2.15-sqlite wagtail_localize.tests.test_edit_translation.TestGetEditTranslationView.test_chooser_in_struct_blocks`